topics/plugin-common.dita

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE task PUBLIC "-//OASIS//DTD DITA Task//EN" "task.dtd">
+<!--  This file is part of the DITA Open Toolkit project. See the accompanying LICENSE file for applicable license.  -->
+<task id="plugin-common">
+  <title>Common plug-in example components</title>
+  <taskbody>
+    <steps>
+      <step id="install-plugin">
+        <cmd>Install the plug-in.</cmd>
```

Comment:
This naturally depends on how you've been developing the plug-in. If you just created a custom plug-in directory under `$DITA_HOME/plugins`, you can run `dita install`. If you have your plug-in code somewhere else, you need to ZIP it and then run `dita install $ZIP_PATH`.

Reply:
Right. Might make sense to standardize this and re-use the instructions as you suggest here. 

So far, the other topics in this section all just use verbatim copies of this sentence.

I'd be inclined to recommend the `configuration.properties` approach that we suggest earlier.